src/time_integration/methods_SSP.jl

```diff
@@ -216,7 +244,7 @@ end
 # stop the time integration
 function terminate!(integrator::SimpleIntegratorSSP)
     integrator.finalstep = true
-    empty!(integrator.opts.tstops)
+    #empty!(integrator.opts.tstops)
```

Comment:
?

Reply:
This was temporarily commented out because the function `empty!()` is not defined for `BinaryHeap`s. I fixed this in 4322f5b5. https://github.com/trixi-framework/Trixi.jl/blob/4322f5b53ec6b56c96114405a8f82553d8b335e5/src/time_integration/methods_SSP.jl#L235-L237